gammapy/image/plotting.py

```diff
@@ -92,33 +89,33 @@ def colormap_hess(vmin, vmax, vtransition, width=0.1):
     return cmap
 
 
-def colormap_milagro(vmin, vmax, vtransition, width=0.0001, huestart=0.6):
+def colormap_milagro(transition=0.5, width=0.0001, huestart=0.6):
     """Colormap often used in Milagro collaboration publications.
 
-    This colormap is gray below ``vtransition`` and similar to the jet colormap above.
+    This colormap is gray below ``transition`` and similar to the jet colormap above.
 
     A sharp gray -> color transition is often used for significance images
-    with a transition value of ``vtransition ~ 5`` or ``vtransition ~ 7``,
+    with a transition value of ``transition ~ 5`` or ``transition ~ 7``,
     so that the following effect is achieved:
 
     - gray: non-significant features are not well visible
-    - color: significant features at the detection threshold ``vmid``
+    - color: significant features at the detection threshold ``transition``
 
     Note that this colormap is often critizised for over-exaggerating small differences
     in significance below and above the gray - color transition threshold.
 
+    The transition parameter is defined between 0 and 1. To calculate the value
+    from data units an `~astropy.visualization.ImageNormalize` instance should be
```

Comment:
Does this sphinx link work ... I thought `ImageNormalize` is only exposed in the `mpl_normalize` sub-namespace.
Can you check the html docs locally?